package.json

```diff
@@ -18,5 +18,8 @@
      "devDependencies": {
```

Comment:
Having some functional issues when I test the app:
1. Not able to send messages with this version
1. When I start the chat there are empty chat bubbles
1. Think it would preferable to aim for the flow that we create a new chat bubble when the other person starts speaking